app/views/event/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<a href="<%=new_event_path%>">Create event</a>
+<a href="<%=events_all_path%>">See your created events</a>
+
+<h1>Upcoming Events List:</h1>
+<br>
+<% @new_list.each do |v| %>
+<p>Title : <%= v.title %></p>
+<p>Description : <%= v.description %></p>
+<p>Date of event : <%= v.event_date %></p>
+<p>By : <%= User.find(v.user_id).name%> 
+<br>
+<% if @attendee.where(event_id: v.id).exists?%>
```

Comment:
Your views shouldn't include any logic. `By:` part should be filled with `event.creator` when you implement that.

You should have line 12 inside your events_controller and display the `event.attendees`.